tests/config/bindings/tests.yml

```diff
@@ -1,3 +1,9 @@
+# Supported operations: create, operations and read
```

Comment:
I don't think we've brought in any other components yet, but the bindings as a whole support more than create and the base operations are all included in the test already.

Reply:
yes this is just for now.